### ISSUE TITLE
Will this work on windows with git-bash and a few tweaks?

### ISSUE THREAD

**Issue by fornof** [tanrax/maza-ad-blocking#4]
Hello, I was wondering if this can be used on windows: 
hosts file is here: c:\windows\system32\drivers\etc\hosts
There are probably some other tweaks? 
curl runs, and bash scripts run in git bash.

**Comment by tanrax** [comment]
I don't work with Windows, I don't know if it would be possible. I'm sorry...

**Comment by fornof** [comment]
I'm looking into it: 
https://hub.docker.com/r/sameersbn/bind/ << this can be used instead of dnsmasq possibly as a dns server.

**Comment by fornof** [comment]
It looks like the list of dns forwards like pihole come from https://pgl.yoyo.org/adservers/serverlist.php?showintro=0&mimetype=plaintext  , This code seems like a wrapper for dnsmasq. Technically, the dns forwards list just needs to be put into the dns server, and the dns server needs to be pointed to in the network.

**Comment by tanrax** [comment]
Great. If it works for you, please let me know. So I can include it in the README

**Comment by fornof** [comment]
Hello tanrax,
I tried BIND dns and it sortof works but has a bunch of configuration, so it sort of works, then I found there is a docker container of pihole which means it can run on anything. I tried it and it works in a docker container on windows just fine! 
[pihole docker](https://hub.docker.com/r/pihole/pihole/)
steps: 
(prereq: [install docker]( https://www.docker.com/products/docker-desktop)) 
1. setup your docker-compose.yml file with the one listed on pihole page (starts with version: '3'). 
2. save and do ``docker-compose up -d ``
3. do  ``docker ps`` and ensure your pihole is running. 
4. Go to network settings and set your DNS to 127.0.0.1 and ::1[ like this](https://mayakron.altervista.org/wikibase/show.php?id=AcrylicWindows10Configuration): 
5. if the docker container is ever stopped, you will need to reverse the setup step 4 to get back internet.
